### PR TITLE
Support for Long Press gesture in CustomLocationViewController

### PR DESCRIPTION
**Pull request by milos-o**
- Adds logic for adding location using long press recogniser

### CODE REVIEW

**Review by nikolamajcen — CHANGES_REQUESTED** [review]
Current code is good 👍 

I would suggest adding notice above latitude and longitude fields with info like; "Long press to change location".

Also, it would be amazing if you can add the pin as well to the selected location.

**Comment by milos-o** [comment]
I have made the changes you mentioned, pin and label for information.

File: Sentinel/Classes/CustomLocation/CustomLocationViewController.swift [hunk 1]
```diff
@@ -67,28 +69,9 @@ class CustomLocationViewController: UIViewController {
...
-        }
-        locationProvider?.setCustomLocation(location: CLLocation(latitude: latitude, longitude: longitude))
-        view.endEditing(true)
-        showAlert(
-            title: "Custom location changed",
-            message: "To apply changes, please restart the application.",
-            actionTitle: "OK",
-            delay: .long
-        )
+        setLocation()
```
[the remark this answers is not in the document]
Comment (nikolamajcen) [reply Sentinel/Classes/CustomLocation/CustomLocationViewController.swift:74]: Additionally, the custom location should be changed when a user taps on "Update", while long pressing is used to change the coordinates.

I would suggest changing the logic behind it a bit, so the long press action just selected the new coordinates, while the "Update" button really sets (applies) the custom location.

**Review by nikolamajcen — CHANGES_REQUESTED** [review]

**Review by nikolamajcen — APPROVED** [review]
All good 👍

**Pull request merged**